client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -220,3 +220,73 @@
 		max-width: 634px;
 	}
 }
+
+.purchase-detail__google-voucher-code {
+	display: flex;
+	align-items: flex-start;
+	margin-top: 12px;
+	padding-bottom: 12px;
+	border-bottom: solid 1px #E9EFF3;
+}
+
+// Google Vouche section
+.purchase-detail__google-voucher-advice {
+	margin-top: 12px;
+	display: flex;
+	max-width: 100%;
+}
+
+.dialog.card.google-voucher-dialog {
```

Comment:
nop, it can't. We need add a stronger rule here.

![image](https://cloud.githubusercontent.com/assets/77539/15774286/d8bed038-297b-11e6-9899-3e5b2f0c0e52.png)